deployment/remote_runner.py

```diff
@@ -0,0 +1,11 @@
+from modal import Image, App, Period
+
+from deployment.agent import run
+
+image = Image.from_registry("ghcr.io/gnosis/prediction-market-agent:main").pip_install("pydantic==2.6.1")
```

Comment:
```suggestion
image = Image.from_registry("ghcr.io/gnosis/prediction-market-agent:main")
```

We have pydantic installed there, don't we?

Reply:
![image](https://github.com/gnosis/gnosis-labs-zuberlin2024/assets/4126235/6d9f54d1-8f4a-44d6-8a45-73ad361f4463)
Pydantic required as extra install, otherwise build fails.

Reply:
I was able to remove that in the end since I run `poetry install` using the project's `pyproject.toml`. So marking this as solved.